<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chip8Class</class>
 <widget class="QMainWindow" name="Chip8Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chip8</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenGame"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStartEmulator"/>
   <addaction name="actionPauseEmulator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpenGame">
   <property name="icon">
    <iconset resource="chip8.qrc">
     <normaloff>:/Chip8/resources/floppy8.svg</normaloff>:/Chip8/resources/floppy8.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Open Game...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="chip8.qrc">
     <normaloff>:/Chip8/resources/machine2.svg</normaloff>:/Chip8/resources/machine2.svg</iconset>
   </property>
   <property name="text">
    <string>Settings...</string>
   </property>
   <property name="toolTip">
    <string>Open Settings dialog</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="chip8.qrc">
     <normaloff>:/Chip8/resources/zoom26.svg</normaloff>:/Chip8/resources/zoom26.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="toolTip">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="chip8.qrc">
     <normaloff>:/Chip8/resources/zooming1.svg</normaloff>:/Chip8/resources/zooming1.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionStartEmulator">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="chip8.qrc">
     <normaloff>:/Chip8/resources/play23.svg</normaloff>:/Chip8/resources/play23.svg</iconset>
   </property>
   <property name="text">
    <string>Start Emulator</string>
   </property>
  </action>
  <action name="actionPauseEmulator">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="chip8.qrc">
     <normaloff>:/Chip8/resources/pause20.svg</normaloff>:/Chip8/resources/pause20.svg</iconset>
   </property>
   <property name="text">
    <string>Pause Emulator</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="chip8.qrc"/>
 </resources>
 <connections/>
</ui>
